packages/cli/snapshots/integration/generators/controller.integration.snapshots.js

```diff
@@ -131,7 +133,7 @@ export class ProductReviewController {
   })
   async findById(
     @param.path.number('id') id: number,
-    @param.query.object('filter', getFilterSchemaFor(ProductReview)) filter?: Filter<ProductReview>
+    @param.query.object('filter', getFilterExcludingWhereSchemaFor(ProductReview)) filter?: FilterExcludingWhere<ProductReview>
```

Comment:
Have you considered using the `options` argument of `getFilterSchemaFor` and `@param.filter`, instead of introducing new functions?

I am concerned that if we need to add another modifier in the future (e.g. "exclude inclusion of related models"), then we will have to introduce four new APIs:
 - `getFilterSchemaExcludingInclusionFor`
 - `getFilterSchemaExcludingWhereExcludingInclusionFor`
 - `@param.filterExcludingInclusion`
 - `@param.filterExcludingWhereExcludingInclusion`

If we add yet another modifier, then we will have to add 8 more APIs, and so on.

I am thinking about the following use:

```suggestion
    @param.query.object('filter', getFilterFor(ProductReview, {excludeWhere: true})) filter?: FilterExcludingWhere<ProductReview>
```